cfn.yaml

```diff
@@ -49,12 +49,18 @@ Resources:
               Action:
                 - lambda:InvokeFunction
               Resource: "*"
-        - PolicyName: PrivateBucket
+        - PolicyName: UnEncryptedPrivateBucket
           PolicyDocument:
             Statement:
                Effect: Allow
                Action: s3:GetObject
                Resource: !Sub arn:aws:s3:::membership-private/${Stage}/salesforce-message-handler.private.conf
```

Comment:
I left the old permission to make sure the lambda can still run in between the cfn is updated and the new code is deployed, I will remove this in another PR